library/deleteDrug.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
shouldn't use caps in dir or filenames (only exception is for classes)

Reply:
updated the case.